### PR TITLE
BR radiomics: generalize methods to run on accession containers 

### PR DESCRIPTION
**Pull request by aauker**
For the breast project, images and labels were added to accessions, not scans.  

- Generalize radiomics code to operate on accession nodes as containers. 
- Overload to run on a single container with `/mind/api/v1/methods/{cohort_id}/{method_id}/{container_id}/run`
- Includes import fix for Node.

### CODE REVIEW

File: data_processing/scanManager/extractRadiomics.py [hunk 1]
```diff
@@ -50,9 +50,9 @@ def add_container_data(container_id, n_meta):
     conn = Neo4jConnection(uri=os.environ["GRAPH_URI"], user="neo4j", pwd="password")
 
     res = conn.query(f""" 
-        MATCH (sc:scan) WHERE id(sc)={container_id}
+        MATCH (object) WHERE id(object)={container_id}
```
Comment (aauker) [inline data_processing/scanManager/extractRadiomics.py:53]: Would like to come up with a better query variable name here, "container"?

Comment (doori) [reply data_processing/scanManager/extractRadiomics.py:53]: container sounds good!

File: data_processing/scanManager/scanManager.py [hunk 2]
```diff
@@ -172,7 +177,13 @@ def post(self, cohort_id, method_id):
...
 
         method_config = res[0].data()['me']
-        scan_ids = requests.get(f'http://{HOSTNAME}:{PORT}/mind/api/v1/scans/{cohort_id}').json()
+
+        a = request.args.get('id', None)
+
+        if container_id==None:
+            scan_ids = requests.get(f'http://{HOSTNAME}:{PORT}/mind/api/v1/scans/{cohort_id}').json()
+        else:
+            scan_ids = [container_id]
```
Comment (doori) [inline data_processing/scanManager/scanManager.py:186]: minor comment. `scan_ids` can be renamed t o `container_ids`

**Review by doori — APPROVED** [review]
Looks great!

**Pull request merged**